### PR TITLE
Stop writing classes

### PR DESCRIPTION
**Pull request by invfo**

### CODE REVIEW

**Comment by invfo** [comment]
ping @ndelon

File: patrick_polbos_orchids/crawler.py [hunk 1]
```diff
@@ -1,47 +1,20 @@
...
-        file.close()
-
+def get_orchids_info(base_url):
+    orchids = get_orchid_page_urls(
+        parse_html(
+            get_html_from_scrollable_page(base_url)
+        )
+    )
+    for orchid in orchids:
+        get_orchid_info(orchid)
```
Comment (ndelon) [inline patrick_polbos_orchids/crawler.py]: It looks unusual that get_orchid_info actually return nothing.
I would see something like this:
- get_orchid_page_urls returns a simple url (just like the function name indicate) and not an Orchid object
- get_orchid_info takes an URL as argument and return an Orchid instance

File: patrick_polbos_orchids/page.py [hunk 2]
```diff
@@ -7,87 +7,73 @@
...
-    def get_temperature(self):
-        return self._get_info_from_table(0)
-
-    def get_luminosity(self):
-        return self._get_info_from_table(1)
+def get_html_from_scrollable_page(url):
+    browser = webdriver.Chrome()
+    browser.get(url)
+    time.sleep(1)
+    SCROLL_PAUSE_TIME = 2
```
Comment (ndelon) [inline patrick_polbos_orchids/page.py]: If this is a kind of constant, it should be defined in the top of the module file.

File: patrick_polbos_orchids/page.py [hunk 3]
```diff
@@ -7,87 +7,73 @@
...
+
+def get_orchid_luminosity(soup):
+    return _get_info_from_table(soup, 1)
+
+
+def get_orchid_info(orchid):
+    soup = parse_html(get_html_from_page(orchid.url))
+    orchid.temperature = get_orchid_temperature(soup)
+    orchid.luminosity = get_orchid_temperature(soup)
+    print(orchid.name, orchid.temperature, orchid.luminosity)
```
Comment (ndelon) [inline patrick_polbos_orchids/page.py]: print here ?

File: patrick_polbos_orchids/page.py [hunk 4]
```diff
@@ -7,87 +7,73 @@
...
+def get_orchid_page_urls(soup):
+    orchid_links = [a for a in soup.find('ul', id='product_list').select('h3 > a')]
+    orchids = []
+    for a in orchid_links:
+        href = a.get('href', '')
+        title = a.get('title', '')
+        if href != '' and title != '' and title != 'Certificats CITES / Phytosanitaire':
+            orchids.append(Orchid(url=href, name=title))
+    return orchids
+
```
Comment (ndelon) [inline patrick_polbos_orchids/page.py]: As told in another comment, I think that this function should simply return urls.
Also: you could probably use yield.

**Comment by invfo** [comment]
@ndelon all your remarks taken into account!

**Pull request merged**